src/app/models/students.model.ts

```diff
@@ -0,0 +1,14 @@
+export class Student {
+  constructor (
+    // public avatar: string,
```

Comment:
remove or if it is going to be used add TODO comment

Reply:
Removed.